src/tensorflow.cc

```diff
@@ -1309,6 +1317,24 @@ AutoCompleteHelper::FixBatchingSupport()
     }
   }
 
+  // Turn on dynamic batching if no batching support is supplied by the
```

Comment:
I believe you must fix the comment here.

Reply:
Updated comment to match the code movement.